components/index/CardsMonitoring/MonitoringItemsOverview/Table/InfectionStatus.vue

```diff
@@ -10,77 +10,56 @@
         />
       </div>
     </li>
+    <li :class="[$style.box]">
+      <div :class="$style.content">
+        <span>
+          {{
+            $t('(2)#7119（東京消防庁救急相談センター）における発熱等相談件数 ')
+          }}
+        </span>
+        <value-with-translatable-unit
+          :value="
+            items[
+              '(2)#7119（東京消防庁救急相談センター）における発熱等相談件数 '
+            ].value
+          "
+          :unit="
+            items[
+              '(2)#7119（東京消防庁救急相談センター）における発熱等相談件数 '
+            ].unit
+          "
+          :bold="
+            items[
+              '(2)#7119（東京消防庁救急相談センター）における発熱等相談件数 '
+            ].bold
+          "
+        />
+      </div>
+    </li>
     <li :class="[$style.box, $style.parent]">
       <div :class="$style.content">
-        <span>{{ $t('潜在・市中感染') }}</span>
+        <span>{{ $t('(3)PCR・抗原検査') }}</span>
       </div>
       <ul :class="$style.group">
         <li :class="[$style.box]">
           <div :class="$style.content">
-            <span>
-              {{
-                $t(
-                  '(2)#7119（東京消防庁救急相談センター）における発熱等相談件数 '
-                )
-              }}
-            </span>
+            <span>{{ $t('陽性率') }}</span>
             <value-with-translatable-unit
-              :value="
-                items[
-                  '(2)#7119（東京消防庁救急相談センター）における発熱等相談件数 '
-                ].value
-              "
-              :unit="
-                items[
-                  '(2)#7119（東京消防庁救急相談センター）における発熱等相談件数 '
-                ].unit
-              "
-              :bold="
-                items[
-                  '(2)#7119（東京消防庁救急相談センター）における発熱等相談件数 '
-                ].bold
-              "
+              :value="items['(4)PCR・抗原検査（陽性率）'].value"
+              :unit="items['(4)PCR・抗原検査（陽性率）'].unit"
+              :bold="items['(4)PCR・抗原検査（検査人数）'].bold"
```

Comment:
```suggestion
              :bold="items['(4)PCR・抗原検査（陽性率）'].bold"
```
こちらは元々の実装がミスってましたね。実装上、表示に変化はないのですが、念のためこちらでコミットしますね。